src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -71,17 +71,21 @@ public void arcadeDrive(double throttle, double turnModifier){
   }
 
   public void curvatureDrive(double throttle, double curvature) {
-    if(throttle != 0.0 && curvature == 0.0 && !gyroDisabled) { //says if straight is zero and turn is not zero
+    if(throttle != 0.0 && curvature == 0.0 && !gyroDisabled) { //driving straight and no turn
       if(this.desiredAngle == Integer.MAX_VALUE) { //means robot just started driving straight
         this.desiredAngle = gyroscope.getGyroAngle(); 
       }
       curvature = this.getAngularError(desiredAngle); 
       this.setLeftSpeed(-(throttle - curvature));
       this.setRightSpeed(throttle + curvature);
     }
-    else {
+    else if(throttle == 0.0 && curvature != 0.0){ // if robot is turning in place 
+      this.setLeftSpeed(-(throttle - curvature));
+      this.setRightSpeed(throttle + curvature);
+    }
+    else { // when robot isn't driving straight
       this.desiredAngle = Integer.MAX_VALUE; //if turn is greater than 0 or if robot is still
-      differentialDrive.curvatureDrive(throttle, curvature, Math.abs(throttle) < QUICK_TURN_THROTTLE_DEADZONE);
+      differentialDrive.curvatureDrive(-throttle, curvature, false);
```

Comment:
I think we need the Boolean check for quick turn if throttle is under .2

Reply:
I think that case is covered by her new else if check.